build/repo.props

```diff
@@ -1,23 +1,7 @@
-<Project>
-  <ItemGroup>
-    <PackageLineup Include="Internal.AspNetCore.Universe.Lineup" Version="2.1.0-*" />
-    <PackageLineup Include="Internal.AspNetCore.Partners.Lineup" Version="2.1.0-*" />
-  </ItemGroup>
-
+﻿<Project>
   <PropertyGroup>
-    <NuGetPublishFeed>https://dotnet.myget.org/f/aspnetcore-ci-dev</NuGetPublishFeed>
+    <!-- These properties are use by the automation that updates dependencies.props -->
+    <LineupPackageId>Internal.AspNetCore.Universe.Lineup</LineupPackageId>
+    <LineupPackageRestoreSource>https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json</LineupPackageRestoreSource>
   </PropertyGroup>
-
-  <!--
-    If this build has a .deps dir supplied by the CI system, use that (and *only* that)
-    // to obtain the package lineup. This ensures we're building and testing against the
-    // specific Coherence build that this Templating repo build is chained to. 
-  -->
-  <PropertyGroup Condition="Exists('$(RepositoryRoot).deps\lineups')">
-    <PolicyRestoreSources>$(RepositoryRoot).deps\lineups</PolicyRestoreSources>
-  </PropertyGroup>
-  <ItemGroup Condition="Exists('$(RepositoryRoot).deps\build')">
-    <AdditionalRestoreSources Include="$(RepositoryRoot).deps\build" />
-  </ItemGroup>
```

Comment:
Could you briefly describe how this stuff is being achieved now?

Reply:
For local, non-orchestrated builds, this repo has the [sources.props](https://github.com/aspnet/templating/blob/namc/merge-dev/build/sources.props) file that point to myget, and package versions are determined by [dependencies.props](https://github.com/aspnet/templating/blob/namc/merge-dev/build/dependencies.props). 

Restore sources and package versions can be overridden by a build orchestrator. Our orchestration is in aspnet/Universe, which will build this repo by invoking `build.cmd /p:DotNetRestoreSourcePropsPath=(generated_file) /p:DotNetPackageVersionPropsPath=(another_generated_file)`

 See https://github.com/dotnet/source-build/blob/dev/release/2.0/Documentation/auto-dependency-flow/api.md for more details